cmd/testscript/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"clickhouse-playground/cmd/testscript/playgroundclient"
+	"clickhouse-playground/cmd/testscript/processor"
+	"flag"
+	"fmt"
+	zlog "github.com/rs/zerolog/log"
+	"time"
+)
+
+type Action string
+
+const (
+	RunTest    Action = "run-test"
+	ImportRuns        = "import-runs"
+)
+
+func main() {
+	var action string
+	var needHelp bool
+	var importRunsBefore string
+	var importRunsAfter string
+	var importResultFilePath string
+
+	flag.BoolVar(&needHelp, "help", false, "help")
```

Comment:
When we agree on the name and purpose of the arguments, we should make a more accurate description of the arguments. 

Imagine you have never used this tool, and you want to measure the efficiency of the playground. The argument description list should be enough for you to understand how the tool works. At the same it should be compact. Not an easy task :)